actionners/kubernetes/tcpdump/tcpdump.go

```diff
@@ -153,7 +153,7 @@ func (a Actionner) Run(event *events.Event, action *rules.Action) (utils.LogLine
 
 	ephemeralContainerName := fmt.Sprintf("%v%v", baseName, uuid.NewString()[:5])
 
-	err = client.CreateEphemeralContainer(pod, containers[0], ephemeralContainerName, defaultTTL)
+	err = client.CreateEphemeralContainer(pod, containers[0], ephemeralContainerName, "issif/tcpdump:latest", defaultTTL)
```

Comment:
can we have this image as a parameter of the action with default pointing to this one? Users in big enterprises might not be able to pull external images that are not approved in their registries.

Reply:
good point